src/views/TechnicalAssistance/RequestForm/Feedback.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Grid, GridContainer, IconArrowBack } from '@trussworks/react-uswds';
+import { sortBy } from 'lodash';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import PageHeading from 'components/PageHeading';
+import Divider from 'components/shared/Divider';
+import { GetTrbRequest_trbRequest as TrbRequest } from 'queries/types/GetTrbRequest';
+import { TRBFeedbackAction } from 'types/graphql-global-types';
+import { formatDateLocal } from 'utils/date';
+
+import Breadcrumbs from '../Breadcrumbs';
+
+type FeedbackProps = {
+  request: TrbRequest;
+  taskListUrl: string;
+};
+
+function Feedback({ request, taskListUrl }: FeedbackProps) {
+  const { t } = useTranslation('technicalAssistance');
+
+  const returnUrl = `/trb/requests/${request.id}/check`;
+
+  const returnToFormLink = useMemo(
+    () => (
+      <UswdsReactLink to={returnUrl}>
+        <IconArrowBack className="margin-right-1 text-middle" />
+        <span className="line-height-body-5">
+          {t('requestFeedback.returnToForm')}
+        </span>
+      </UswdsReactLink>
+    ),
+    [returnUrl, t]
+  );
+
+  return (
+    <GridContainer className="width-full">
+      <Breadcrumbs
+        items={[
+          { text: t('heading'), url: '/trb' },
+          {
+            text: t('taskList.heading'),
+            url: taskListUrl
+          },
+          {
+            text: t('requestForm.heading'),
+            url: returnUrl
+          },
+          { text: t('requestFeedback.viewFeedback') }
+        ]}
+      />
+
+      <PageHeading className="margin-top-6- margin-bottom-1">
+        {t('requestFeedback.heading')}
+      </PageHeading>
+      {returnToFormLink}
+
+      {sortBy(
+        request.feedback.filter(
+          e => e.action === TRBFeedbackAction.REQUEST_EDITS
+        ),
+        'createdAt'
+      )
+        .reverse()
+        .map(item => {
+          return (
+            <React.Fragment key={item.id}>
+              <Grid row gap className="margin-top-4 margin-bottom-6">
+                <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
+                  <dl className="easi-dl">
+                    <dt>{t('requestFeedback.date')}</dt>
+                    <dd data-testid="feedback-date">
+                      {formatDateLocal(item.createdAt, 'MMMM d, yyyy')}
+                    </dd>
+                  </dl>
+                </Grid>
+                <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
+                  <dl className="easi-dl">
+                    <dt>{t('requestFeedback.from')}</dt>
+                    <dd>{item.author.commonName}</dd>
```

Comment:
In Figma the feedback author is listed as "FirstName Lastname, TRB".

<img width="131" alt="Screenshot 2023-02-03 at 3 25 07 PM" src="https://user-images.githubusercontent.com/60187543/216713949-92243ac5-4ab3-4607-b57f-753030687ac4.png">

Reply:
Only the name is supplied from the datapoint being used

Reply:
I know only the UserInfo type is supplied from the data point.

What I'm saying is that based on the Figma, the name should be formatted with "TRB" after it in plain text. I'm assuming it applies to all authors because the feedback is coming from a TRB admin.

Reply:
I see, thank you! I wasn't sure if it could've been a dynamic label. I'll make it static.